### ISSUE TITLE
Migrate to Zipkin 2.0

### ISSUE THREAD

**Issue by jcchavezs** [Vinelab/tracing-laravel#26]
Zipkin PHP 2.0 is out and brought support for PHP 7.1+ avoiding type checks with almost no breaking changes. Consider upgrading.

**Comment by adiachenko** [comment]
@jcchavezs  Sure, thanks for the reminder. I noticed from tests that tracer stopped propagating "x-b3-flags" header by default. Is that something that we should worry about? 

From my understanding, this flags correlates to isDebug property on TraceContext although I am not too sure what it does or how to enable it since we don't directly manage this object.

**Comment by jcchavezs** [comment]
> I noticed from tests that tracer stopped propagating "x-b3-flags" header by default. Is that something that we should worry about? 

Debug should only be propagated if it is `1` (see https://github.com/openzipkin/b3-propagation#debug-flag) and we tried to honor that in the test https://github.com/openzipkin/zipkin-php/commit/091cc933debe42be5b3dbe4fe5e5e66f7a58a946#diff-71cf0cce040421feb2f4f628b86a03f9b0d8587b949820f9b506d31356aa2e47L258. What changed is that if `x-b3-flags:1` we mark the context as sampled too (but when injecting the context we only pass the sampling flag if `isDebug` is false.

TL;DR from this library perspective we should not worry about it as propagation will handle it (unless you found a case).
